example/scalalib/native/1-simple/build.mill

```diff
@@ -0,0 +1,56 @@
+//// SNIPPET:BUILD
```

Comment:
We don't need these `SNIPPET` blocks since we're not extending or being extended by other examples